app/src/main/java/com/lastaoutdoor/lasta/data/db/ActivitiesDBRepositoryImpl.kt

```diff
@@ -84,6 +84,9 @@ constructor(context: Context, database: FirebaseFirestore) : ActivitiesDBReposit
     if (!query.isEmpty) {
       val documents = query.documents
       for (document in documents) {
+        if (document.get("startPosition") == null) {

```

Comment:
I believe that this check isn't necessary and **should** be **removed** as the startPosition would never be null, when it's not valid it is always (0, 0) in the database

Reply:
you are right, I don't know how it got into my code since I don't remember writing it, but I also found that weird when I saw it, and thought it was someone else who had done this voluntarily. Will remove it now